### ISSUE TITLE
Can't make --finddupes to find a dupe

### ISSUE THREAD

**Issue by danaivehr** [shirosaidev/diskover#72]
diskover is installed from docker hub linuxserver/diskover
diskover version: v1.5.0.7
Pulled from linuxserver/diskover just yesterday

Before start it on the big file storage i'm trying to test it on the small test location.

Maybe i'm missing something, so please point me out how to deal with it.

diskover is up and running, indexed all files. I can go to myhost:80 and search files, browse files and tag them.
The only thing i can't make to work is **duplicates** search.

Just to prove i have at least two dupes:
* i open a card of each dupe file in diskover and seeing the same Filehash: 7bc14d8b88b05d5b199246dfd4977e26
* files are in different directories
* files has the same name and same date
* only differences are access / change time (not modify time) and inode number
* Dupe MD5 field is empty
* screenshots are attached here
![2019-11-03_173652](https://user-images.githubusercontent.com/1226019/68086812-9f091980-fe60-11e9-8f40-2b4478c229b1.png)
![2019-11-03_173714](https://user-images.githubusercontent.com/1226019/68086813-a29ca080-fe60-11e9-880e-8c618073d845.png)

To run dupes search i've tryed:
1. Run {"action": "finddupes", "index": "diskover-index"} on the web Admin page - nothing happens except terminal message
`2019-11-03 17:41:13,840 [INFO][diskover] [thread-3]: Got command from ('172.29.4.4', 59500) `
2. Go to Analytics -> Dupes on the web diskover: just got
`Sorry, no duplicate files found. Run diskover using --finddupes to search for duplicate files, or try searching for a different directory and clicking it's dupes analytics button, or changing filters (on filetree page) or mindupes value.`
3. Run in terminal `/app/diskover/diskover.py -d /data -i diskover-index --finddupes`, getting:
```                                                                                                                                                                                                                                                                                                               
                                                                                                                                                                                                                                                                                                               
     _/_/_/    _/            _/                                                                                                                                                                                                                                                                                
    _/    _/        _/_/_/  _/  _/      _/_/    _/      _/    _/_/    _/  _/_/                                                                                                                                                                                                                                 
   _/    _/  _/  _/_/      _/_/      _/    _/  _/      _/  _/_/_/_/  _/_/                                                                                                                                                                                                                                      
  _/    _/  _/      _/_/  _/  _/    _/    _/    _/  _/    _/        _/                                                                                                                                                                                                                                         
 _/_/_/    _/  _/_/_/    _/    _/    _/_/        _/       _/_/_/  _/                                                                                                                                                                                                                                           
                              v1.5.0.7                                                                                                                                                                                                                                                                         
                              https://shirosaidev.github.io/diskover                                                                                                                                                                                                                                           
                              "I didn't even know that was there."                                                                                                                                                                                                                                             
                              Support diskover on Patreon or PayPal :)                                                                                                                                                                                                                                         
                                                                                                                                                                                                                                                                                                               
                                                                                                                                                                                                                                                                                                               
2019-11-03 17:44:50,790 [INFO][diskover] Using config file: /app/diskover/diskover.cfg                                                                                                                                                                                                                         
2019-11-03 17:44:50,810 [INFO][diskover] Found 8 diskover RQ worker bots                                                                                                                                                                                                                                       
2019-11-03 17:44:50,811 [INFO][diskover] Searching diskover-index for all duplicate files...                                                                                                                                                                                                                   
2019-11-03 17:44:50,834 [INFO][diskover] 0 file hashes have been enqueued                                                                                                                                                                                                                                      
| Checking (Queue: 0) Elapsed Time: 0:00:00                                                                                                                                                                                                                                                                    
2019-11-03 17:44:50,848 [INFO][diskover] DONE checking for dupes! Sayonara!
```

I can provide more details, if needed.

**Comment by shirosaidev** [comment]
https://github.com/shirosaidev/diskover/wiki/Duplicate-files-(dupes)

Try without -d

**Comment by danaivehr** [comment]
The same without -d
`0 file hashes have been enqueued`

**Comment by shirosaidev** [comment]
can you run with --debug --finddupes and attach the output.

**Comment by shirosaidev** [comment]
what is your maxsize setting in dupescheck section in diskover.cfg ?

**Comment by danaivehr** [comment]
Okay, i don't know python, but just digg into diskover_dupes.py

Pasting some logger.info() in several places i found out:
it never comes to 
```
for hit in res['hits']['hits']:                                                                                                                                                                                                                                                                        
            filehash = hit['_source']['filehash']
```

because res['hits']['hits'] is emty.
And res['hits']['hits'] is empty because of filter in data.
After i commented out filter by type : file, filehashes were queued:

```
    data = {                                                                                                                                                                                                                                                                                                   
        "size": 0,                                                                                                                                                                                                                                                                                             
        "query": {                                                                                                                                                                                                                                                                                             
            "bool": {                                                                                                                                                                                                                                                                                          
                "must": {                                                                                                                                                                                                                                                                                      
                    "term": {"hardlinks": 1}                                                                                                                                                                                                                                                                   
                },                                                                                                                                                                                                                                                                                             
#                "filter": {                                                                                                                                                                                                                                                                                   
#                    "term": {"type": "file"}                                                                                                                                                                                                                                                                  
#                },                                                                                                                                                                                                                                                                                            
                "must_not": {                                                                                                                                                                                                                                                                                  
                    "range": {                                                                                                                                                                                                                                                                                 
                        "filesize": {                                                                                                                                                                                                                                                                          
                            "gt": config['dupes_maxsize'],                                                                                                                                                                                                                                                     
                            "lt": cliargs['minsize']                                                                                                                                                                                                                                                           
                        }                                                                                                                                                                                                                                                                                      
                    }                                                                                                                                                                                                                                                                                          
                }                                                                                                                                                                                                                                                                                              
            }                                                                                                                                                                                                                                                                                                  
        }                                                                                                                                                                                                                                                                                                      
    }                                                                                                                                                                                                                                                                                                          

```

Does it means that crawler wrote wrong data to index? How can i check it?

**Comment by danaivehr** [comment]
> what is your maxsize setting in dupescheck section in diskover.cfg ?

default 
maxsize = 1073741824

**Comment by shirosaidev** [comment]
try replacing your diskover_dupes.py with this one https://github.com/shirosaidev/diskover/blob/master/diskover_dupes.py

I've fixed a bug in it that got introduced in v1.5.0.7.

**Comment by danaivehr** [comment]
Hurray, it worked!
Thank you very much for your product, i'm really enjoing it.
